src/Core/src/Internal/Factory.php

```diff
@@ -82,6 +85,7 @@ public function make(string $alias, array $parameters = [], string $context = nu
         } finally {
             /** @psalm-var class-string $alias */
             $this->state->bindings[$alias] = $binding;
+            $this->trace = [];
```

Comment:
What about case when the `FactoryInterface` or `ContainerInterface` are used in a dependency constructor?